### PR TITLE
Create reducers and actions with ducks pattern

### PR DESCRIPTION
**Pull request by Javivy**
## In this milestone I implemented the following functions and enhancements:

  - [x] Created Reducers with [Ducks Pattern](https://github.com/erikras/ducks-modular-redux)
  - [x] Created Actions with [Ducks Pattern](https://github.com/erikras/ducks-modular-redux)
  - [x] Configured store with `redux-toolkit`

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const CHECK_STATUS = 'CHECK_STATUS';
+
+// Constants
+const defaultState = [];
+
+// Reducer
+export default function categoriesReducer(state = defaultState, action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return console.log('Under Construction');
```
Comment (Whoistolu) [inline src/redux/categories/categories.js]: - [ ] [OPTIONAL] Kindly remove this `console` statement as it is only meant for debugging. This sometimes makes the application slower and it is also not a good practice to always leave it in your codebase. You can return the 'Under Construction' statement without using `console.log`

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable arrow-body-style */
```
Comment (Whoistolu) [inline src/redux/categories/categories.js]: - [ ] [REQUIRED] There are too many eslint-disable. Kindly enable `eslint` and fix the errors manually as it checks the code standard and helps us write consistent code with good quality.

File: src/redux/books/books.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable arrow-body-style */
```
Comment (Whoistolu) [inline src/redux/books/books.js]: - [ ] [REQUIRED] There are too many eslint-disable. Kindly enable `eslint` and fix the errors manually as it checks the code standard and helps us write consistent code with good quality.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Javivy 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [REQUIRED] There are too many eslint-disable. Kindly enable `eslint` and fix the errors manually as it checks the code standard and helps us write consistent code with good quality.  We have more in /components/Book.js, /components/DisplayBooks.js and the /components/Navbar.js. i have stated these routes here because they are not in your file changes.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @Javivy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**